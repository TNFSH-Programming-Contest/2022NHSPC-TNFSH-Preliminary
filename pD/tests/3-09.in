7 7
3 0 2 1 1 2 1
2 1 1 1 1 2 2
T....T.
......T
..T...T
T......
.....T.
.T.....
...T..T
