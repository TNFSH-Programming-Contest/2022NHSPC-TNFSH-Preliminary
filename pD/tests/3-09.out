TA..ATA
......T
A.T..AT
T.A....
.....TA
AT.A...
...T.AT
